include/SFML/System/Vector2.hpp

```diff
@@ -22,10 +22,10 @@
 //
 ////////////////////////////////////////////////////////////
 
+#include <cmath>
 #ifndef SFML_VECTOR2_HPP
 #define SFML_VECTOR2_HPP
```

Comment:
Look at other headers that have `#include` statements to match the style used there. Headers aren't included before header guards and there is a special comment block used before the first `#include` statement.